lib/scrabble_word.rb

```diff
@@ -0,0 +1,27 @@
+class ScrabbleWord
+
+  attr_reader :word
+
+  def initialize word
+    @word = word.downcase
+  end
+
+# Calculate score.
+
+  def score
+    score = 0
+    score += @word.count( 'aeioulnrst' ) * 1
```

Comment:
How could you DRY this up so that you only need to write `score += @word.count` once?